sdks/python/apache_beam/transforms/combiners_test.py

```diff
@@ -392,5 +394,86 @@ def test_global_fanout(self):
       assert_that(result, equal_to([49.5]))
 
 
+class LatestTest(unittest.TestCase):
+
+  def test_globally(self):
+    l = [window.GlobalWindows.windowed_value(3, 100),
```

Comment:
Consider using `TimestampedValue` instead of `windowed_value`? I think it will make the code easier to understand. Same below.